gnc/sonar/nodes/image_pub.cpp

```diff
@@ -0,0 +1,104 @@
+#include <bvt_sdk.h>
+#include <ros/ros.h>
+#include <ros/console.h>
+#include <opencv2/core/core.hpp> 
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.h>
+
+#include <ctime>
+
+#include "BVWrapper.h"
+
+
+int main(int argc, char** argv){
+	try{
+		ros::init(argc, argv, "sonar_image_publisher");
+		ros::NodeHandle nh;
+		image_transport::ImageTransport it(nh);
+		image_transport::Publisher pub_bw = it.advertise("sonar/mono8", 1);
+		image_transport::Publisher pub_bw16 = it.advertise("sonar/mono16", 1);
+		image_transport::Publisher pub_color = it.advertise("sonar/color", 1);
+
+		const char host[] = "192.168.37.53";
```

Comment:
Do you think the blueview sdk will accept a hostname? It might want the IP internally for some reason and not be happy with only the hostname.